govcd/monitor.go

```diff
@@ -130,6 +130,14 @@ func prettyTask(task *types.Task) string {
 	return ""
 }
 
+func prettyEdgeGatewayServiceConfiguration(conf *types.EdgeGatewayServiceConfiguration) string {
```

Comment:
Please add comment that this returns json as itsn't mentioned in func name.

Reply:
Done